Library/ViewModels/ErroredBackingViewViewModel.swift

```diff
@@ -37,9 +45,65 @@ public final class ErroredBackingViewViewModel: ErroredBackingViewViewModelType,
     self.manageButtonTappedObserver.send(value: ())
   }
 
+  public let finalCollectionDateText: Signal<String, Never>
   public let notifyDelegateManageButtonTapped: Signal<GraphBacking, Never>
   public let projectName: Signal<String, Never>
 
   public var inputs: ErroredBackingViewViewModelInputs { return self }
   public var outputs: ErroredBackingViewViewModelOutputs { return self }
 }
+
+private func timeLeftString(date: String) -> String {
+  let dateFormatter = ISO8601DateFormatter()
```

Comment:
We should cache date formatters as they're pretty expensive to create so may have an adverse effect on performance.

This can be done similarly to how we cache our formatters in [Format.swift](https://github.com/kickstarter/ios-oss/blob/master/Library/Format.swift#L408).